downgrade

```diff
@@ -271,6 +271,7 @@ main() {
   for term; do
     if ! process_term "$term"; then
       {
+        term=$(sed -r "s/(.*)\b(>=|<=|<|>|=)\b(.*)/\1/g" <<<"$term")
```

Comment:
Right now, this operator-version "parsing" happens exactly once, within `process_term`. If we start doing it in more places, we run the risk of it going out of sync. For such a minor, not-happy-path feature like this error message, I wouldn't to it, personally.

If you want to keep it, that's fine. I might also consider the following two options:

1. Print this error message from within `process_term`, before it returns `1`, where `$name` is already available
2. Extract this `sed` to a function and re-use it

Reply:
Somehow I am still clinging on to this error message, because I feel like if it gives you an output with one of the operator symbols, it should already be telling you that it was unable to parse the operator. If it is fine with you, I will keep this and follow your suggestions.

Reply:
Took option 1 in this case to keep the `sed` operator parsing to just one place so we don't need to add another function now.